express/code/scripts/mep/ace1057/grid-marquee/grid-marquee.js

```diff
@@ -0,0 +1,294 @@
+    } = await import('../../../utils/pricing.js');
+    const {
+      url,
+      country,
+      language,
+      offerId,
+    } = await fetchPlanOnePlans(a.href);
+    const newTrialHref = buildUrl(url, country, language, getConfig, offerId);
+    a.href = newTrialHref;
+  } catch (error) {
+    window.lana.log(`Failed to fetch prices for page plan: ${error}`);
+  }
+  a.style.visibility = 'visible';
+  return a;
+}
+
+function decorateHeadline(headline) {
+  const ctas = headline.querySelectorAll('a');
+  if (!ctas.length) return headline;
+  ctas[0].parentElement.classList.add('ctas');
+  ctas.forEach((cta) => {
+    cta.classList.add('button');
+    formatDynamicCartLink(cta);
+  });
+  ctas[0].classList.add('primaryCTA');
+  headline.classList.add('headline');
+  return headline;
+}
+
+async function makeRating(store) {
+  const { replaceKey } = await import(`${getLibs()}/features/placeholders.js`);
+  const ratings = (await replaceKey('app-store-ratings', getConfig()))?.split(';') || [];
+  const link = ratings[2]?.trim();
+  if (!link) {
+    return null;
+  }
+  const [score, cnt] = ratings[['apple', 'google'].indexOf(store)].split(',').map((str) => str.trim());
+  const storeLink = createTag('a', { href: link }, getIconElementDeprecated(`${store}-store`));
+  const { default: trackBranchParameters } = await import('../../../branchlinks.js');
+  await trackBranchParameters([storeLink]);
+  return createTag('div', { class: 'ratings-container' }, [score, getIconElementDeprecated('star'), cnt, storeLink]);
+}
+
+function makeRatings() {
+  const ratings = createTag('div', { class: 'ratings' });
+  const userAgent = getMobileOperatingSystem();
+  const cb = (el) => el && ratings.append(el);
+  // eslint-disable-next-line chai-friendly/no-unused-expressions
+  userAgent !== 'Android' && makeRating('apple').then(cb);
+  // eslint-disable-next-line chai-friendly/no-unused-expressions
+  userAgent !== 'iOS' && makeRating('google').then(cb);
+  return ratings;
+}
+
+function createToggle({ toggleText, toggleActive, toggleBypassParam }) {
+  const bypassParam = toggleBypassParam.querySelector('div:nth-child(2)').innerText;
+  const toggleWrapper = createTag('div', { class: 'toggle-wrapper' });
+  const toggleLabels = toggleText.querySelectorAll('li');
+  const isChecked = toggleActive.querySelector('div:nth-child(2)')?.innerText || 1;
+
+  toggleText.remove();
+  toggleBypassParam.remove();
+  toggleActive.remove();
+  if (document.location.href.includes(bypassParam)) return null;
+
+  toggleWrapper.innerHTML = `
+    <span class="toggle_label_unchecked" daa-ll="Individual vs Business toggle">${toggleLabels[0]?.innerText}</span>
+    <label class="toggle" daa-ll="Individual vs Business toggle">
+      <input daa-ll="Individual vs Business toggle" type="checkbox" ${isChecked ? 'checked' : ''}>
+      <span class="slider round"></span>
+    </label>
+    <span class="toggle_label_checked">${toggleLabels[1]?.innerText}</span>
+  `;
```

Comment:
So you can define innerHTML in one step:
```suggestion
  const toggleWrapper = createTag('div', { class: 'toggle-wrapper' },
  `
    <span class="toggle_label_unchecked" daa-ll="Individual vs Business toggle">${toggleLabels[0]?.innerText}</span>
    <label class="toggle" daa-ll="Individual vs Business toggle">
      <input daa-ll="Individual vs Business toggle" type="checkbox" ${isChecked ? 'checked' : ''}>
      <span class="slider round"></span>
    </label>
    <span class="toggle_label_checked">${toggleLabels[1]?.innerText}</span>
  `);
```
will need linter clean up